src/styles/components/_card.scss

```diff
@@ -0,0 +1,164 @@
+@import "../abstracts/_variables.scss";
+@import "../base/base";
+
+@media only screen and (max-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(1, 1fr);
+    max-width: $card-width;
+    margin: 0 auto;
```

Comment:
`margin: 0 auto;` this is common for all media queries, also we keep media queries next to the styles that they concern. In sass (scss) media queries can be nested inside the class that they apply to.